src/abstractdataframe/iteration.jl

```diff
@@ -440,15 +444,33 @@ julia> mapcols(x -> x.^2, df)
    2 │     4    144
    3 │     9    169
    4 │    16    196
+
+julia> mapcols(x -> x.^2, df, cols="y")
```

Comment:
Maybe something which looks more useful like this?
```suggestion
julia> mapcols(x -> x.^2, df, cols=r"y")
```